presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -263,6 +263,15 @@ Property Name                                      Description
 ``hive.file-status-cache-expire-time``             How long a cached directory listing should be considered     ``1m``
                                                    valid.
 
+``hive.parquet.time-zone``                         Adjusts binary timestamp values to a specific time zone.     JVM default  
+                                                   For Hive 3.1+, this should be set to UTC. 
+
+``hive.rcfile.time-zone``                          Adjusts binary timestamp values to a specific time zone.     JVM default  
```

Comment:
For here I think we want to say `binary encoded` since that is different from the default rc encoding which is text.